pkg/collector/corechecks/servicediscovery/module/comm_test.go

```diff
@@ -59,8 +59,11 @@ func TestIgnoreComm(t *testing.T) {
 
 // TestIgnoreCommsLengths checks that the map contains names no longer than 15 bytes.
 func TestIgnoreCommsLengths(t *testing.T) {
-	for comm := range ignoreComms {
-		assert.LessOrEqual(t, len(comm), 15, "Process name %q too big", comm)
+	discovery := newDiscovery()
+	require.NotEmpty(t, discovery)
+
+	for comm := range discovery.config.ignoreComms {
```

Comment:
if len(discovery.config.ignoreComms) == 0; then you don't test anything here

Reply:
Right, this test checks length of commands if configuration has commands. There is another unit test verifying length of configuration parameter and it's match to expected

Reply:
no, this check aims to verify the default values are properly configured, but you don't verify that you have default values

Reply:
I've added this slice lenght verification